examples/3d-car-coin-hunt/3d-car-coin-hunt.json

```diff
@@ -4276,6 +4396,943 @@
+                      },
+                      "parameters": [
+                        "Icon",
+                        "Tween",
+                        "\"Rotate\"",
+                        "-8",
+                        "\"easeInOutQuad\"",
+                        "2",
+                        ""
+                      ]
+                    }
+                  ]
+                },
+                {
+                  "type": "BuiltinCommonInstructions::Standard",
+                  "conditions": [
+                    {
+                      "type": {
+                        "value": "CompareObjectTimer"
+                      },
+                      "parameters": [
+                        "Object",
+                        "\"Animate\"",
+                        ">=",
+                        "4"
+                      ]
+                    },
+                    {
+                      "type": {
+                        "value": "BuiltinCommonInstructions::Once"
+                      },
+                      "parameters": []
+                    }
+                  ],
+                  "actions": [
+                    {
+                      "type": {
+                        "value": "ResetObjectTimer"
+                      },
+                      "parameters": [
+                        "Object",
+                        "\"Animate\""
+                      ]
+                    },
+                    {
+                      "type": {
+                        "value": "Tween::TweenBehavior::AddObjectAngleTween2"
+                      },
+                      "parameters": [
+                        "Icon",
+                        "Tween",
+                        "\"Rotate\"",
+                        "8",
+                        "\"easeInOutQuad\"",
+                        "2",
+                        ""
+                      ]
+                    }
+                  ]
+                }
+              ],
+              "parameters": [
+                {
+                  "description": "Object",
+                  "name": "Object",
+                  "supplementaryInformation": "ScreenOrientationChecker::ScreenOrientationChecker",
+                  "type": "object"
+                }
+              ],
+              "objectGroups": []
+            },
+            {
+              "fullName": "",
+              "functionType": "Action",
+              "name": "onCreated",
+              "sentence": "",
+              "events": [
+                {
+                  "type": "BuiltinCommonInstructions::Standard",
+                  "conditions": [],
+                  "actions": [
+                    {
+                      "type": {
+                        "value": "ResetObjectTimer"
+                      },
+                      "parameters": [
+                        "Object",
+                        "\"Animate\""
```

Comment:
I think it's better to use children. I'm not sure the parent object will still work the same in the future.

Also, the timer is probably unnecessary as tweens has "is finished" condition, so you can loop between 2 tweens directly.

Reply:
Done with "is finished", but I also had to remove tweens, it was painful to find how to do it.